### DISCUSSION TITLE
Require Hovering to expand the vertical tabs bar

### DISCUSSION THREAD

**Discussion by cswimr** [zen-browser/desktop#1888] [Ideas]
Not really sure what the best way to explain this is, but I'll use this video comparing Zen and Microsoft Edge's behavior.

https://github.com/user-attachments/assets/92c13b78-d5c0-49a2-b283-3843976532d3

It's not immediately clear what I'm doing in that video, but I'm moving my mouse from the browser window to my second monitor. In Zen, this makes the tab bar expand, which is very jarring. But Microsoft Edge has a short delay between whenever you hover over the tab bar and when it actually expands, so it doesn't look nearly as jarring. I think this being the default behavior (or just an option!) in Zen would be a big improvement.

Also, on the topic of things that Edge does that I think Zen should also do, Edge has a short "forgiveness" period between when you stop hovering over the extended tab bar and when it collapses, depending on how far away your cursor is from the tab bar. I think this would also be nice, especially for people who use higher mouse sensitivities or are using less precise input devices like touchpads.

**Comment by gumbarros** [comment]
![image](https://github.com/user-attachments/assets/f4ee1fec-7ab3-439d-b36a-6b928d8354b7)

**Reply by Corrodias** [reply]
Tell them what? It seems that this project is still under active development.

**Reply by gumbarros** [reply]
> Tell them what? It seems that this project is still under active development.

It was literally removed, I understand the project is under development, but removing a core feature and moving to a plugin feels very bad.
